From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 28 Sep 2019 08:11:49 +0200
Subject: [PATCH] make _PATH_CONNERRS world readable

There is nothing security-sensitive there.


Imported from ppp_2.4.7-2+4.1.debian.tar.xz

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 pppd/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pppd/main.c b/pppd/main.c
index 8e31365f0c58..ed544315c1df 100644
--- a/pppd/main.c
+++ b/pppd/main.c
@@ -1673,7 +1673,7 @@ device_script(program, in, out, dont_wait)
     if (log_to_fd >= 0)
 	errfd = log_to_fd;
     else
-	errfd = open(_PATH_CONNERRS, O_WRONLY | O_APPEND | O_CREAT, 0600);
+	errfd = open(_PATH_CONNERRS, O_WRONLY | O_APPEND | O_CREAT, 0644);
 
     ++conn_running;
     pid = safe_fork(in, out, errfd);
